Fun Commands/Mini-Games/Dyno-RPG/rpg-help.txt

```diff
@@ -0,0 +1,45 @@
+{delete}
+{silent}
+{require: Adventurer}
+**__Avelon RPG Commands__**
```

Comment:
Is this the Avelon RPG or the Dyno RPG? I'm starting to become a bit confused on that tbh.

Reply:
Fixed sorry about that lol